beaker-vue/src/components/cell/BeakerQueryCellEvent.vue

```diff
@@ -97,11 +96,12 @@
 
             </div>
         </div>
-        <span v-else-if="props.event?.type === 'code_cell'">
+        <div v-else-if="props.event?.type === 'code_cell'" style="position: relative;">
```

Comment:
These changes are not exactly "leaving intact" the BeakerQueryCellEvent, but they're also harmless. Actually, changing the span to a div makes more sense if we want to explicitly work with the box-block model and not an inline container (such as spans to contain other complex elements), since that causes different subtle bugs across browsers.